src/core/tests/Test_StatusHandler.py

```diff
@@ -397,6 +397,30 @@ def test_sort_packages_by_classification_and_state(self):
             self.assertEqual(installation_patches_sorted[12]["name"], "test-package-2")  # | Other              | Excluded    |
             self.assertEqual(installation_patches_sorted[13]["name"], "test-package-1")  # | Other              | NotSelected |
 
+    def test_if_complete_status_is_empty_json_with_bad_name(self):
```

Comment:
_test_if_status_file_resets_on_load_if_malformed_

better describes what is actually being validated to be true.